<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modify_produits</class>
 <widget class="QDialog" name="modify_produits">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Products Modification &amp; Delete</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255,255,255);</string>
  </property>
  <widget class="QPushButton" name="b_supprimer_produits">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(158,59,59);
font: 75 9pt &quot;Arial&quot;;
border: 2px solid rgb(158,59,59);
border-radius: 20px;
}

QPushButton:hover {
  color: #fff; /* Change text color on hover */
  background-color: #a63b3b; /* Change background color on hover */
  border-color: #a63b3b; /* Change border color on hover */
}

QPushButton:pressed {
  color: #fff; 
  background-color: #802c2c; 
  border-color: #802c2c; 
}</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_prix_u_produits">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 1px solid;
border-color: #9B0000;
border-radius: 10px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_id_u_produits">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 1px solid;
border-color: #9B0000;
border-radius: 10px;
}</string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="b_modifier_produits">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(158,59,59);
font: 75 9pt &quot;Arial&quot;;
border: 2px solid rgb(158,59,59);
border-radius: 20px;
}

QPushButton:hover {
  color: #fff; /* Change text color on hover */
  background-color: #a63b3b; /* Change background color on hover */
  border-color: #a63b3b; /* Change border color on hover */
}

QPushButton:pressed {
  color: #fff; 
  background-color: #802c2c; 
  border-color: #802c2c; 
}</string>
   </property>
   <property name="text">
    <string>Modify</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_nom_u_produits">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 1px solid;
border-color: #9B0000;
border-radius: 10px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_quantity_u_produits">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 1px solid;
border-color: #9B0000;
border-radius: 10px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>331</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressources.qrc">:/new/prefix1/gfx/3516835.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>b_supprimer_produits</zorder>
  <zorder>le_prix_u_produits</zorder>
  <zorder>le_id_u_produits</zorder>
  <zorder>b_modifier_produits</zorder>
  <zorder>le_nom_u_produits</zorder>
  <zorder>le_quantity_u_produits</zorder>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
